workorders.lic

```diff
@@ -260,15 +260,22 @@ class WorkOrders
   def shape_items(info, item, quantity)
     ensure_copper_on_hand(5000, @hometown)
     recipe, items_per_stock, spare_stock, scrap = find_recipe(info, item, quantity)
+    stock_name = 'stock-name'
+    stock_number = 'stock-number'
 
     quantity.times do |count|
-      if (count % items_per_stock).zero?
+      if items_per_stock.zero? || (count % items_per_stock).zero?
         dispose("#{info['stock-name']} lumber") if count > 0 && spare_stock
```

Comment:
Should this be `dispose("#{info[stock_name]} lumber") if count > 0 && spare_stock`?

Reply:
Fixing that ended up breaking the script, and it's not something I changed from the original though